src/components/TextInputFocusable/index.js

```diff
@@ -160,6 +163,7 @@ class TextInputFocusable extends React.Component {
      */
     checkForAttachment(event) {
         const {files, types} = event.clipboardData;
+        const {translations: {translate}} = this.props;
```

Comment:
Why do this? Is it just to not have to write `this.props.translations.translate`?